network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -75,7 +75,10 @@ void test() {
         try (NetworkStoreService service = createNetworkStoreService(randomServerPort)) {
             // import new network in the store
             assertTrue(service.getNetworkIds().isEmpty());
-            Network network = service.importNetwork(new ResourceDataSource("test", new ResourceSet("/", "test.xiidm")));
+            Properties properties = new Properties();
+            properties.setProperty("ucte.import.create-areas", "false");
```

Comment:
You could have a static variable and use it everywhere to avoid to repeat this code